lib/lita/handlers/urban_dictionary.rb

```diff
@@ -44,8 +48,10 @@ def fetch_definition(term)
       end
 
       def message_from_lines(lines, term)
-        if lines.size > 20
-          lines = lines[0..19]
+        # strip out tabs and spaces baked into lines repsonse
+        lines = lines.map {|line| line.split(/[\r\n]+/)}.flatten
```

Comment:
Let's move this logic into its own method so the comment isn't needed. Use `flat_map` instead of `map` + `flatten` for improved performance. For style, put a space after `{` and before `}`.